static/app/views/performance/newTraceDetails/traceDrawer/details/span/index.tsx

```diff
@@ -473,6 +475,17 @@ function EAPSpanNodeDetails({
 
                     <LogDetails />
 
+                    {organization.features.includes('trace-view-span-links') &&
+                      links?.length && (
+                        <TraceSpanLinks
+                          node={node}
+                          links={links}
+                          theme={theme}
+                          location={location}
+                          organization={organization}
+                        />
+                      )}
```

Comment:
nit: Thoughts on swapping out the "rendering" `&&` to a ternary, protecting against accidentally rendering a zero once the flag is removed

```suggestion
                    {(organization.features.includes('trace-view-span-links') &&
                      links?.length) ? (
                        <TraceSpanLinks
                          node={node}
                          links={links}
                          theme={theme}
                          location={location}
                          organization={organization}
                        />
                      ) : null}
```

Reply:
Oop never mind, I was reading something else! Yes, fixed 👍🏻